### PR TITLE
Solution for Issue #21

### PR DESCRIPTION
**Pull request by Rubikan**
I added support to change the ghostscript executable since I ran into problems using it on windows. The problem could easily be circumvented using a symlink, but it is nice to have a built-in solution for that.

### CODE REVIEW

**Comment by Rubikan** [comment]
Wow ... I managed to totally forget about this since my need for it at work was on halt. Sorry for that!

@jonmagic yeah, the proposal made perfectly sense, I didn't match my new configuration to the already existing code and I understand that you would like it to match.

File: README.textile [hunk 1]
```diff
@@ -63,6 +63,16 @@ We also support using other processors (the default is whatever version of Image
   pdf = Grim.reap('/path/to/pdf)
 </code></pre>
 
+It is also supported to use another ghostscript executable, which is especially interesting for windows users since the executable is named differently there.
+
+<pre><code>
+  # specifying another ghostscript executable, win64 in this example
+  # the ghostscript/bin folder still has to be in the PATH for this to work
+  Grim.processor =  Grim::ImageMagickProcessor.new({:ghostscript_executable => "gswin64c.exe"})
```
Comment (jnunemaker) [inline README.textile]: This is out of date now.

**Comment by Rubikan** [comment]
Yeah ... I could have thought of that, sorry!

**Comment by jnunemaker** [comment]
Quick skim seems good to me. Up to @jonmagic the rest of the way.

**Comment by jonmagic** [comment]
I just want to run the tests quick before merging. I'll be working on OSS tomorrow so expect an update then :)

**Comment by jonmagic** [comment]
@Rubikan can you let me know what version rspec is set to in your Gemfile.lock? Silly me didn't specify a version in Gemfile so it installed the newest when I ran bundle and I'm getting all sorts of errors since the `should` syntax has been deprecated.

Side note: I'm going to work on a branch today that updates all of the specs to use the newer `expect` syntax and remove all of the old `should` stuff. I'll also lock rspec in the Gemfile :)

**Comment by Rubikan** [comment]
Is it possible that I another one other thing to this PR? 
Its about the possibility to use the alpha option from imagemagick, of course I'll add a test for it too.

Or should I do another PR after you merged this one?

**Comment by jonmagic** [comment]
> Is it possible that I another one other thing to this PR? 
> Its about the possibility to use the alpha option from imagemagick, of course I'll add a test for it too.
> 
> Or should I do another PR after you merge this one?

I think it makes sense to do in a second PR... I'm going to merge this now, sorry it took so long.

**Comment by jonmagic** [comment]
I can wait to cut a new release until you put out the second PR, or I can do a release now and then a release then. @Rubikan if you think you'll get a new PR cut soon I'll wait, if not let me know and I'll go ahead and release :) Also, just updated the [contributors list with your name](https://github.com/jonmagic/grim/commit/874ea8962cb15c68e74ef6ed1361f6b78c0214fc)!

**Comment by Rubikan** [comment]
The new PR is already up :) #24

**Pull request merged**